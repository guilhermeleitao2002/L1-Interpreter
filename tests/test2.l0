let a = true;
let b = false;
let andResult = a && b;
let orResult = a || b;
let notResult = ~a;
let x = 10;
let y = 5;
let result = if (x > y) {
  println x;
  x
} else {
  println y;
  y
};
println andResult;
println orResult;
println notResult;
println result;;