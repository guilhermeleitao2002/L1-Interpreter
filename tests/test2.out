false
true
false
x is greater than y
10